### ISSUE TITLE
node-marked-man: testsuite failures against node 10 and node-marked 0.5.1

### ISSUE THREAD

**Issue by LocutusOfBorg** [kapouer/marked-man#19]
```
node test/compare.js
Test failure, result written in /node-marked-man-0.3.0/test/man/links.err
Test failure, result written in /node-marked-man-0.3.0/test/man/markdown_syntax.err
Test failure, result written in /node-marked-man-0.3.0/test/man/table.err
Test failure, result written in /node-marked-man-0.3.0/test/man/underline_spacing_test.err
Test failure, result written in /node-marked-man-0.3.0/test/out/links.err
Test failure, result written in /node-marked-man-0.3.0/test/out/entity_encoding_test.err
Test failure, result written in /node-marked-man-0.3.0/test/out/definition_list_syntax.err
Test failure, result written in /node-marked-man-0.3.0/test/out/nested_list_with_code.err
Test failure, result written in /node-marked-man-0.3.0/test/out/markdown_syntax.err
Test failure, result written in /node-marked-man-0.3.0/test/out/underline_spacing_test.err
troff: <standard input>:173: warning [p 1, 2.0i, div '3tbd17,3', 0.0i]: cannot adjust line
troff: <standard input>:272: warning [p 1, 2.0i, div '3tbd28,3', 0.2i]: cannot adjust line
troff: <standard input>:671: warning [p 1, 2.0i, div '3tbd73,0', 0.0i]: cannot adjust line
warning: file '<standard input>', around line 18:
  table wider than line width
warning: file '<standard input>', around line 13:
  table squeezed horizontally to fit line length

Test failure, result written in /node-marked-man-0.3.0/test/out/table.err
Failed tests:  7
Succeeded tests:  5
make[1]: *** [Makefile:7: check] Error 1

```

@kapouer can you please have a look? this is also tracked as debian bug #912168 and making a lot of npm packages look sad...

**Comment by LocutusOfBorg** [comment]
@kapouer ping... this is preventing nodejs migration and freeze is coming...

**Comment by kapouer** [comment]
Will do it today. Bog me tomorrow if it wasn't.

**Comment by kapouer** [comment]
Fixed in 0.4.0 (0.4.2 is the same as 0.4.0).
